### PR TITLE
Fixed #25385 -- Allowed importing views.generic.View from views.View.

### PR DESCRIPTION
**Pull request by varunsharma**
Changes in views/__init__.py, docs and release notes

### CODE REVIEW

File: docs/releases/1.9.txt [hunk 1]
```diff
@@ -377,6 +377,10 @@ Generic Views
   or tuple of decorators. It can also be used to :ref:`decorate classes instead
   of methods <decorating-class-based-views>`.
 
+* The :class:`~django.views.generic.base.View` class can now also be imported like::
```
Comment (timgraham) [inline docs/releases/1.9.txt]: Move to 1.10 release notes and use this language:
```
can be imported from ``django.views``.
```

File: django/views/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.views.generic.base import View
+
+all = ['View']
```
Comment (timgraham) [inline django/views/__init__.py]: `__all__ = `

**Comment by timgraham** [comment]
Probably worth a mention `docs/ref/class-based-views/base.html` with a `.. versionchanged:: 1.10` annotation. Could modify a test file just to verify the import works, e.g. `view_tests/views.py:from django.views.generic import View`

**Comment by varunsharma** [comment]
Done with the improvements. Please review.

**Comment by timgraham** [comment]
buildbot, add to whitelist.

**Comment by timgraham** [comment]
merged in 3d6474e1a50dbaf9684f7fb34bfccb77630f27be, thanks!

**Pull request closed without merging**